annlite/core/codec/pq.py

```diff
@@ -36,7 +37,7 @@ def __init__(
         dim: int,
         n_subvectors: int = 8,
         n_clusters: int = 256,
-        metric: Metric = Metric.EUCLIDEAN,
+        metric: Metric = Metric.COSINE,
```

Comment:
this seems to be a very big breaking change right?

Reply:
Yeah, have met with Felix: `Annlite` default uses cosine as the distance metric, and `pq_codec` needs to be aligned while training.